src/model/beneficial.owner.other.model.ts

```diff
@@ -8,16 +8,21 @@ export const DateKey: string = "startDate";
 export const DateKeys: string[] = ["startDate-day", "startDate-month", "startDate-year"];
 
 export const BeneficialOwnerOtherKey: string = "beneficialOwnerOther";
-export const BeneficialOwnerOtherKeys: string[] = ["corporationName", "principalAddress", "isSameAddress", "serviceAddress", "lawGoverned", "startDate", "natureOfControl", "statementCondition", "isSanctioned"];
+export const BeneficialOwnerOtherKeys: string[] = ["corporationName", "principalAddress", "isSameAddress", "serviceAddress", "legalForm", "lawGoverned", "registerName", "registerNumber", "publicRegister", "startDate", "natureOfControlIndividual",  "natureOfControlTrust", "natureOfControlFirm", "isSanctioned"];
 
 export interface BeneficialOwnerOther {
-    corporationName?: string
-    principalAddress?: Address
-    isSameAddress?: yesNoResponse
-    serviceAddress?: Address
-    lawGoverned?: string
-    startDate?: InputDate
-    natureOfControl?: natureOfControl
-    statementCondition?: statementCondition
-    isSanctioned?: yesNoResponse
+    corporationName?: string;
+    principalAddress?: Address;
+    isSameAddress?: yesNoResponse;
+    serviceAddress?: Address;
+    legalForm?: string;
+    lawGoverned?: string;
+    registerName?: string;
+    registerNumber?: string;
+    publicRegister?: yesNoResponse;
+    startDate?: InputDate;
+    natureOfControlIndividual?: [NatureOfControlType];
```

Comment:
should be `natureOfControlIndividual?: NatureOfControlType[];` and same for the others, I am fixing what we have got on the individual BO model

Reply:
actually they should be 
  beneficial_owner_nature_of_control_types?: NatureOfControlType[];
  trustees_nature_of_control_types?: NatureOfControlType[];
  non_legal_firm_members_nature_of_control_types?: NatureOfControlType[];
  
  to match the api model naming as there is no conversion done in SDK